### ISSUE TITLE
First time when activity loads it shows all the view (overlapping)

### ISSUE THREAD

**Issue by rakshitsoni02** [Nightonke/WoWoViewPager#4]
and when i swipe it works fine please help me resolve this issue as i am using your given sample of cvactivity.[https://drive.google.com/file/d/0B5ObQNL9tgDkSVk2ajZpVUdtRzA/view?usp=sharing](url)

**Comment by Nightonke** [comment]
You mean, when you use the cvactivity that I write, this bug happens? Did the cvactivity be changed? Thanks for reporting.

**Comment by rakshitsoni02** [comment]
exactly! and i didn't changed cvactivity using as it is from sample.

**Comment by Nightonke** [comment]
That's so weird. But I will check this.

**Comment by microcian** [comment]
Facing Same issue

**Comment by microcian** [comment]
after scanning barcode, I get Apk , In that application no overlapping issue, so i think there must be difference between apk and source code

**Comment by mohanadmohie** [comment]
Such a great library, but I'm having the same problem. The APK works fine, but the sample doesn't. Has anyone managed to solve it?

**Comment by adjulbic** [comment]
viewPager.setAdapter(adapter);
viewPager.post(new Runnable() {
      @Override public void run() {
        viewPager.setCurrentItem(0);
      }
    });

Setting current item to 0 after setting adapter resolved an issue for me.

**Comment by kiaanpillay** [comment]
Hi, having the exact same problem. If I scroll a little bit it goes away though. Any fix?

**Comment by kamerok** [comment]
Fixed this by adding initial animation
```
private void initView() {
        ViewAnimation animation = new ViewAnimation(view);
        animation.addPageAnimaition(new WoWoAlphaAnimation(
                0, 0, 0,
                0,
                0,
                EaseType.Linear,
                false
        ));
        animation.addPageAnimaition(new WoWoAlphaAnimation(
                1, 0, 1,
                0,
                1,
                EaseType.Linear,
                false
        ));
        animation.addPageAnimaition(new WoWoAlphaAnimation(
                2, 0, 1,
                1,
                0,
                EaseType.Linear,
                false
        ));
        wowo.addAnimation(animation);
    }
```
First animation do the trick
All initializations hapening in `onCreate` method

**Comment by Nightonke** [comment]
Hi all. 
The bug is fixed in version 2.0.0 and above. Just need to call the `wowo.ready()` method after adding animations for WoWoViewPager. Check [Basic Usage Wiki](https://github.com/Nightonke/WoWoViewPager/wiki/Basic-Usage) for more information.
